<html>

<head>
<meta http-equiv="Content-Language" content="en-us">
<meta name="GENERATOR" content="Microsoft FrontPage 5.0">
<meta name="ProgId" content="FrontPage.Editor.Document">
<meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<title>Investigating Polar Bear and Giant Panda Ancestry</title>
<style>
<!--
 p.MsoNormal
	{mso-style-parent:"";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	font-size:11.0pt;
	font-family:Calibri
	}
 table.MsoNormalTable
	{mso-style-parent:"";
	font-size:10.0pt;
	font-family:"Times New Roman"}
span.simplelink
	{}
-->
</style>
</head>

<body>

<p class="MsoNormal" style="line-height: normal"><b>
<span style="font-size: 12.0pt; font-family: Comic Sans MS">Investigating Polar 
Bear and Giant Panda Ancestry (New Version 2009)</span></b></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">(Adapted from Maier, 
C.A. (2001) “Building Phylogenetic Trees from DNA Sequence Data: Investigating 
Polar Bear &amp; Giant Panda Ancestry.” <i>The American Biology Teacher.</i> 63:9, 
Pages642-646.)&nbsp; Adapted for NGWB by Mark Alan Miller January 2009.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">Introduction:</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">This activity will 
allow you to use the tools used by geneticists and evolutionists located and 
maintained by the National Center for Biotechnology Information. These tools 
will allow you to compare the isolated and sequenced genes from species that are 
stored in GenBank. Each unique sequence is identified by an “accession number” 
as unique as your social security number. You will also be able to compare 
sequences and generate phylogenetic trees using Biology Workbench that contains 
specific application such as CLUSTALW (sequence similarity tool), CLUSTALDIST 
(generates a genetic distance matrix) and DRAWGRAM (creates phylogenetic trees). 
The gene sequence that is being used in this activity is the 12s ribosomal RNA 
gene sequence from several bear species and the giant panda. The 12s gene is a 
good candidate for study because it doesn’t undergo recombination (not subject 
to meiosis) because the gene is found on the mitochondrial chromosome* which is 
inherited only from the mother via the egg. </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">* Check out this 
link for a discussion of the use of mitochondrial DNA and if it really is only 
from the mother. <b><i><span style="color: #00007F">Athena Review †Vol.2, no.2</span></i></b><span style="color: #00007F">:
<i>†Recent Finds in Paleoanthropology</i> <b>&nbsp;Molecular clockwork and related 
theories</b></span></span><b><span style="font-size: 24.0pt; font-family: Arial; color: #00007F">
</span></b><span style="font-size: 10.0pt; font-family: Comic Sans MS">
<a href="http://www.athenapub.com/molclock.htm" style="color: blue; text-decoration: underline; text-underline: single">
http://www.athenapub.com/molclock.htm</a></span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">Instructions for 
Accessing 12s rRNA Gene Sequences</span></p>
<p class="MsoNormal" style="line-height: normal"><u>
<span style="font-size: 10.0pt; font-family: Comic Sans MS">PART 1:&nbsp; 
(tutorials found at </span></u><span style="font-size: 10.0pt">&nbsp; </span>
<a HREF="http://www.ngbw.org/labs/bears/bearlab.htm"><u>
<font SIZE="2" COLOR="#0000ff">http://www.ngbw.org/labs/bears/bearlab.htm</font></u></a><u><font SIZE="2" color="#0000FF">&nbsp;
</font></u></p>
<p class="MsoNormal" style="line-height: normal"><font SIZE="2">
<a HREF="http://atol.sdsc.edu/instructions/howto.htm">
<font SIZE="2" COLOR="#0000ff"><u>http://atol.sdsc.edu/instructions/howto.htm</u></font></a></p>
</font>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">a. Open a new web page or 
new tab and Enter:
<a href="http://www.ncbi.nlm.nih.gov" style="color: blue; text-decoration: underline; text-underline: single">
http://www.ncbi.nlm.nih.gov</a></span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">b. In the Search bar; select 
&quot;nucleotide&quot; from the drop down menu</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">c. Enter the accession 
number, to obtain each of the following 12s rRNA gene sequences one at a time, 
and select “GO”&nbsp; to the right of the bar. The information for the bear will 
appear in the window, click on the accession number.</span></p>
<table class="MsoNormalTable" border="1" cellspacing="1" cellpadding="0" width="325" style="width: 243.75pt">
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal"><b>
    <span style="font-size: 10.0pt; font-family: Arial">Species</span></b></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal"><b>
    <span style="font-size: 10.0pt; font-family: Arial">Accession Number</span></b></td>
  </tr>
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">American Black Bear</span></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">Y08520</span></td>
  </tr>
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">American Brown Bear</span></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">L21889</span></td>
  </tr>
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">Spectacled Bear</span></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">L21883</span></td>
  </tr>
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">Asiatic Black Bear</span></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">L21890</span></td>
  </tr>
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">Polar Bear</span></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">L22164</span></td>
  </tr>
  <tr>
    <td width="54%" valign="top" style="width: 54.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">Giant Panda</span></td>
    <td width="46%" valign="top" style="width: 46.0%; padding: 5.25pt">
    <p class="MsoNormal" style="line-height: normal">
    <span style="font-size: 10.0pt; font-family: Arial">Y08521</span></td>
  </tr>
</table>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">e. Find the bar with 
“GenBank” to the right of the “Display” use the drop down menu and select &quot;FASTA&quot; 
to bring up the sequence in FASTA format.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">f. Open a new web page or 
new tab so you can toggle between the two pages and go to: SWAMI, The Next 
Generation Biology Workbench: </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">(
<a href="http://www.ngbw.org/">http://www.ngbw.org/</a>&nbsp;&nbsp; ), set up a 
free account (if you need help;
<a href="http://www.ngbw.org/help/register.htm" style="color: blue; text-decoration: underline; text-underline: single">
http://www.ngbw.org/help/register.htm</a>). Once your registration is complete, 
you will be logged in to your personal area. </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">g. The NGBW allows you to 
store data and tasks in folders just like MS Outlook, or other a mail clients. 
So before working here, you must create at least one folder. Click on the button 
that says “Create New Folder”. Name your folder, and save the name. If you get 
stuck, here is the help link (<a href="http://www.ngbw.org/help/create_folder.htm" style="color: blue; text-decoration: underline; text-underline: single">http://www.ngbw.org/help/create_folder.htm</a>).&nbsp; 
The folder will appear on the left side of the screen.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">h. Each NGBW folder has a 
data area and a tool area. For now, we are going to upload some sequences, so 
click on the data area icon. It will open the Data management page; now click on 
the Upload/Enter Data button.&nbsp; </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">j. Go back to the NCBI page; 
highlight the entire FASTA sequence (including the “&gt;”). Copy the highlighted 
sequence.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">i. Go back the NGBW Data 
Upload page. </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">j. In the “label” box, (at 
the top of the upload form), type in the name of the bear, or any other label 
you will recognize.<br>
<br>
k. Position the cursor in the top left of the data entry box and “Paste” the 
sequence. You will now need to replace the FASTA description with the name of 
the bear, making sure the “&gt;” symbol is still the first symbol. BUT: <b><i>do 
not introduce blank space</i></b>. For example, type American_Black_Bear or 
Americanblackbear, NOT American Black Bear.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">Use the dropdown boxes at 
the&nbsp; bottom of the page to tell the application what kind of data you are 
uploading. The Entity Type is “Nucleic Acid”, the Data Type is “Sequence”, and 
the format is “FASTA”. Now select “SAVE”. You only need to set these once, the 
application remembers after that.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">l. Return to GenBank (on the 
other web page). Copy and Paste the remaining 12s rRNA gene sequences into the 
NGBW in exactly the same way.</span></p>
<p class="MsoNormal" style="line-height: normal"><u>
<span style="font-size: 12.0pt; font-family: Arial">Instructions Part II: 
Analyze the 12s rRNA Gene Sequences</span></u></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">1. <u>Align the five 
different Bear 12s rRNA Gene Sequences (not the panda)</u></span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">a. In the NGBW, click on the 
“Tasks” icon in your working folder. When the Task management pane appear, click 
on the “Create New Task” Button. Enter a description for the task and click the 
“Set Description” button. </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">b. Now click on the “Select 
Input Data” button. Check the click boxes to the left of all the bears (not the 
panda yet). Click the “Select Data” button.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">c. When the Task Creation 
Pane re-appears, click the “Select Tool” button. From the Nucleic Acids Sequence 
Tools tab, choose “CLUSTALW_N” (the tools are alphabetical). Now click the &quot;Save 
and Run Task&quot; button.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">d. A new page will load that 
lets you follow the progress of your jobs. Click the “Refresh Tasks” tab near 
the top of the page, until the “View Status” button on the right turns into 
“View Output.” Click on the “View Output”: tab, and a page showing your results 
will appear. Click on the link “outfile.aln”, and the results of your alignment 
will be exposed. Click the “Save to Current Folder” button.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">e. A window will appear that 
lets you name and specify the kind of data you are saving. Enter a data Label, 
then select Entity Type: “Nucleic Acid” Data Type: “Sequence Alignment” and 
Format: “Clustal”. Then click the “Save” button. </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">2. <u>Determine the Genetic 
Distance Between Sequence Pairs</u></span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">a. Now go back to the Tasks 
area of your folder. Click the “Create New Task” button. Give the task a 
description and “Set the Description”, just like before. Click the “Select Data” 
button, and find your alignment data, check the box to the left of the 
alignment, and click “Select Data.” When the task creation page reloads, choose 
Select Tool, and find “CLUSTALW_DIST” under the Phylogeny/Alignment Tools 
tab.&nbsp;As in step 1.e above, the Entity Type: “Nucleic Acid” Data Type: “Sequence 
Alignment” and Format: “Clustal”. But you shouldn’t have to change anything, the 
application remembers your last entry. Then click the “Save” button. &nbsp;When the 
task creation page appears, click the “Save and Run” button. </span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">b. When the Task management 
page reloads, use the “Refresh Tasks” button to monitor when the job completes. 
When the “View Output” button appears, click on it, and expose the results. 
Click on the infile.dst link to expose the Distance Matrix results. Record the 
distance matrix in your journal to examine later.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">3. <u>Build a Phylogenetic 
Tree of the Bear Species</u></span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">a. CLUSTALW_DIST also 
outputs a phylogenetic tree. Under the “View Output” for CLUSTALW_DIST click on 
the infile.ph link. Save this data to your current folder, just like in 1.d and 
1.e. But this time chose Entity Type: “Taxon” Data Type: “Phylogenetic Tree” and 
Format: “Newick”</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">b. Now go find this data 
item in the Data area of your folder, under the tab “Phylogenetic Trees.” Click 
the data item, and it will open up, revealing these two links “</span><span class="simplelink"><span lang="EN"><a href="http://www.ngbw.org/web/data!display.action?id=48257" style="color: blue; text-decoration: underline; text-underline: single">Show/Hide 
Data Contents</a> |
<a href="javascript:popitup('/web/draw.action?id=48257')" style="color: blue; text-decoration: underline; text-underline: single">
Draw Tree</a>.” </span>
<span lang="EN" style="font-size: 10.0pt; font-family: Arial">Cllck on</span><span lang="EN">
</span><span lang="EN" style="font-size: 10.0pt; font-family: Arial">the Draw 
Tree link, and you will see an interactive view of the Tree.</span></span><span lang="EN">
</span><span style="font-size: 10.0pt; font-family: Arial">Record the diagram in 
your journal.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">4. D<u>etermine the 
relationship of Giant Panda to the Bear Phylogenetic Tree</u></span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">a. In GenBank (if you 
haven’t already) access the giant panda 12s rRNA gene sequence and import it 
into the your NGBW Data area.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Arial">b. Run a new alignment with 
all five bears and the panda sequence. Import the alignment into your data area, 
use “CLUSTALW_DIST” to create a distance matrix and a phylogenetic tree. Import 
the phylogenetic tree into your data area do see the tree. Record all 
information.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">Analysis: (due 
typed)</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">1. Create a table 
showing the distance matrix of the five bears and the panda (Please note that a 
distance of 0.00 indicates identical sequences and as the difference in gene 
sequences increases so does the number).</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">2. Provide a 
phylogenetic tree of the bears and the panda. Label the lines on the tree with 
the corresponding distances.</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">3. Explain and 
provide support for the conclusions made by the program (Biology Workbench).
</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">4. Red Panda are 
reported to be more closely related to raccoon. Find the relationship (provide 
both the distance matrix and tree) between the Red Panda and raccoons to the 
others you’ve already examined. Is the first statement supported?</span></p>
<p class="MsoNormal" style="line-height: normal">
<span style="font-size: 10.0pt; font-family: Comic Sans MS">5. Pick a few 
animals and run their 12s rRNA patterns to see how they are related. Record this 
information.</span></p>
<p class="MsoNormal">&nbsp;</p>

</body>

</html>