<!doctype html public "-//w3c//dtd html 4.0 transitional//en"><html><head>   <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">   <meta name="Generator" content="Microsoft FrontPage 5.0">   <meta name="GENERATOR" content="Mozilla/4.7C-CCK-MCD {C-UDP; EBM-APPLE} (Macintosh; I; PPC) [Netscape]">   <title>Exercise</title><style TYPE="text/css">     /* Created using Style Master from Western Civilisation CSS1 { 49 95 451 409 } */A.offsitelink:link{background-color: silver}    </style></head><body text="#333333" bgcolor="#FFCC99" link="#0033FF" vlink="#FF0000" alink="#0000FF" TOPMARGIN="16" LEFTMARGIN="10" MARGINWIDTH="10" MARGINHEIGHT="16"><center><p><b><font size=+2 face="Times New Roman,Times,Times NewRoman">Generationof Phylogenetic Tree based upon DNA sequence analysis</font></b></center><p><b><font size=+1 face="Times New Roman,Times,Times NewRoman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </font></b>Adapted from<b><font face="Times New Roman,Times,Times NewRoman">Scott Cooper (University of WisconsinóLa Crosse, <a href="mailto:cooper@mail.uwlax.edu">cooper@mail.uwlax.edu</a> )</font></b><p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Originally found at&nbsp; (http://bioquest.org/bioinformatics/module/cooper/Exercise/exercise.htm)<p class="outsidelink">A phylogenetic tree is a type of graph that scientistsuse to classify related organisms.&nbsp; Traditionally these trees werecreated using physical traits of organisms such as bone structure, beakshape, etc.&nbsp; More recently molecular data has been used to supportand refine these phylogenetic trees.&nbsp; This is done by aligning DNAsequences for a particular gene from several organisms and then applyinga program to examine the number of mutations that have accumulated betweenthese sequences.&nbsp; The more accumulated mutations, the more distantlyrelated the two species.<p><b>Exercise:</b><p>There are two steps to creating a phylogenetic tree:<ul>1.&nbsp; Aligning the DNA sequences<p>2.&nbsp; Using the aligned DNA sequences to generate a tree</ul><p><br><b><font color="#990066">Using Biology WorkBench to align two ormore sequences</font>.</b><p><font face="Times New Roman,Times,Times NewRoman">First we will alignDNA sequences from the <i>Mitochondrial D-loop</i> from five species <i>(Human,Chimpanzee (2) include regular chimpanzee and pygmy chimpanzee, Gorilla, Orangutan and Neanderthal)</i>.&nbsp;&nbsp; These sequencescan be found at the site <a href="http://www.ncbi.nlm.nih.gov/">NCLB</a>&nbsp; under &quot;search&quot; all databases select &quot;corenucleotides&quot; and enter the following codes to get nucleotide sequences for each these primates:</font><p><font face="Times New Roman,Times,Times NewRoman">Human: <i>Homo sapiens </i>AB241389</font><p><font face="Times New Roman,Times,Times NewRoman">Neanderthal:<i>Homo sapiens neanderthalensis</i> AF254446</font><p><font face="Times New Roman,Times,Times NewRoman">Chimpanzee (1): <i>Pan troglodytes</i> AJ851169</font><p><font face="Times New Roman,Times,Times NewRoman">Pygmy Chimpanzee (2): <i>Pan paniscus</i> AJ829473</font><p><font face="Times New Roman,Times,Times NewRoman">Gorilla: <i>Gorilla gorilla</i> AJ422244</font><p><font face="Times New Roman,Times,Times NewRoman">Orangutan:<i>Pongo pygmaeus</i> AJ627443</font><p><font face="Times New Roman,Times,Times NewRoman">You can use </font><font face="Arial" size="2"><a href="http://workbench.sdsc.edu">http://workbench.sdsc.edu</a>, or</font><p><font color="#000000">Log onto the </font><a href="http://workbench.sdsc.edu" class="offsitelink">BiologyWorkBench</a><i>,</i><b> (also see <a href="http://bioweb.uwlax.edu/GenWeb/Molecular/Seq_Anal/seq_anal.htm#bw" class="offsitelink">ScottCooper's information about the Biology WorkBench</a>)</b><br>select <b>Session Tools </b><font color="#000000">and<b>create a newsession</b> named <b><i>phylo</i></b>.</font><p><font color="#000000">Select&nbsp; <b>Nucleic Tools</b> and then scrolldown to <b>Add New Sequence</b>.</font><p><font color="#000000">Create files for each species and paste in themitochondrial D-loop sequence.</font><p>Select the sequences that you wish to align by clicking the box nextto each file, and then select <b>CLUSTALW</b>.<p><img SRC="Phylo1.gif" ALT="Phylo1" BORDER=0 id="Picture32" height=371 width=896><p>You will now be given some options on parameters you can change in youralignment.&nbsp; You can just use the default values and select <b>Submit</b>at the bottom of the page.<p>The results will show the five sequences with colored letters representinga consensus.&nbsp; Black letters will illustrate a mismatch and dasheswill represent gaps.<p><img SRC="Align.gif" ALT="Phylo2" BORDER=0 id="Picture31" height=336 width=612><br>&nbsp;<p><font face="Times New Roman,Times,Times NewRoman">Select <b>Import Alignment(s).</b>Thiswill save this alignment in a single file for further analysis.</font><br>&nbsp;<br>&nbsp;<p><b><font color="#990066">Using Biology WorkBench</font> <font color="#990066">tocreate a phylogenetic tree</font>.</b><p><font face="Times New Roman,Times,Times NewRoman">Next select <b>AlignmentTools</b> select the aligned sequences by clicking the box next to thefile name.</font><p>Next click CLUSTALDIST (to get the distances and paste into your lab report, you'll need the distances to put on the DRAWGRAM)<p><font face="Times New Roman,Times,Times NewRoman">Next click eitherthe button <b>DRAWGRAM.</b></font><p><font face="Times New Roman,Times,Times NewRoman"><b>DRAWGRAM</b> drawsa rooted tree.</font><p>You will now be given some options on parameters you can change in youralignment.&nbsp; You can just use the default values and select <b>Submit</b>at the bottom of the page.<p>The tree will now appear.<p><font face="Times New Roman,Times,Times NewRoman">Congratulations youare now an evolutionary biologist.</font><br>&nbsp;<br>&nbsp;<p><b><font face="Times New Roman,Times,Times NewRoman"><font color="#990066">Analysisof trees</font></font></b><p><font face="Times New Roman,Times,Times NewRoman">Examine your tree.</font><p>Draw your tree. Add the distances from CLUSTALDIST information to the tree.<p><font face="Times New Roman,Times,Times NewRoman">Which species appearto be the most closely related?</font> (make sure you use data to support and elaborate your answer.)</p><p><font face="Times New Roman,Times,Times NewRoman">From your tree, doesit appear that Neanderthals were direct ancestors of humans, or did weshare a common ancestor that branched off from the other apes?</font> (make sure you use data to support and elaborate your answer.)<br>&nbsp;<br><font face="Times New Roman,Times,Times NewRoman">Try this analysis with 4-5 other <a href="http://www.ncbi.nlm.nih.gov/Taxonomy/tax.html" class="offsitelink">species</a>of your choosing.</font> ( you can generate one tree with all 10-11 species)<br>&nbsp;<br>Include your clustal distance matrices (clustal matrices and trees may be copied and pasted from Workbench).&nbsp;<p><br>&nbsp;<p><font size=+1 face="Arial,Helvetica">You are currently viewingan archived version of this material.&nbsp;</font><br><font size=+1 face="Arial,Helvetica">For the current BiologyStudent Workbench Project please visit&nbsp;</font><br><font size=+1 face="Arial,Helvetica"><a href="http://bioquest.org/bioinformatics/" class="offsitelink">BioQUEST/bioinformatics</a>or <a href="http://peptide.ncsa.uiuc.edu/" class="offsitelink">BiologyStudent Workbench</a></font> </p><p>Off-site linksóthose that require an Internet connectionóare highlighted(e.g., <a href="http://bioquest.org" class="offsitelink">BioQUEST.org</a>).<p><font size=+1 face="Arial,Helvetica">This exercise is also availablefrom the author at the <a href="http://bioweb.uwlax.edu/GenWeb/genweb.html" class="offsitelink">GenWeb</a>site.</font><p><font size=+1 face="Arial,Helvetica">Return to the <a href="../../">BSWModule Index</a></font></body></html>