<html>

<head>
<meta http-equiv="Content-Language" content="en-us">
<meta name="GENERATOR" content="Microsoft FrontPage 5.0">
<meta name="ProgId" content="FrontPage.Editor.Document">
<meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
<title>Student Instructions for Restriction Digest</title>
<style>
<!--
 p.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:"Times New Roman";
	margin-left:0in; margin-right:0in; margin-top:0in}
 table.MsoNormalTable
	{mso-style-parent:"";
	font-size:10.0pt;
	font-family:"Times New Roman"}
 li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:"Times New Roman";
	margin-left:0in; margin-right:0in; margin-top:0in}
-->
</style>
</head>

<body>

<p class="MsoNormal"><span lang="RU"><br>
<![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></p>
<p class="MsoNormal">Student Instructions for Restriction Digest (NEB)</p>
<p class="MsoNormal"><b>
<span lang="RU" style="font-size: 16.5pt; font-family: Times">DNA Restriction 
Enzyme Mapping</span><span lang="RU" style="font-size: 6.0pt; font-family: Times">1</span></b></p>
<p class="MsoNormal">
<span lang="RU" style="font-size: 10.0pt; font-family: Times">S2501, 
Contemporary Biology Laboratory</span><span style="font-size: 10.0pt; font-family: Times">, 
Columbia</span></p>
<p class="MsoNormal">
<span lang="RU" style="font-size: 10.0pt; font-family: Times">Summer 2009</span></p>
<p class="MsoNormal">
<span lang="RU" style="font-size: 7.5pt; font-family: Times">1</span><span lang="RU" style="font-size: 10.0pt; font-family: Times">Adapted 
from D. A. Micklos and G. A. Freyer,&nbsp;<i>DNA Science, a First Course in 
Recombinant DNA Technology,</i></span><span lang="RU" style="font-size: 7.5pt; font-family: Times">
</span><span lang="RU" style="font-size: 10.0pt; font-family: Times">Cold Spring 
Harbor Laboratory Press<i>,&nbsp;</i>Cold Spring Harbor, N.Y., 1990, ch. 3, and pp. 
247-275.</span></p>
<p class="MsoNormal"><span style="font-size: 16.5pt; font-family: Times">&nbsp;</span></p>
<p class="MsoNormal"><b><span lang="RU" style="font-family: Times">I. 
Introduction</span></b></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">Molecular 
biology is a hybrid discipline that arose from a confluence of disciplines:</span><span style="font-family: Times">
</span><span lang="RU" style="font-family: Times">genetics, physical chemistry, 
X-ray crystallography, biochemistry, microbiology, bacteriology, and</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">virology. It represents the 
antithesis of the notion that physics, chemistry, and biology are separate and 
unrelated divisions of the natural sciences.In the beginning of the 20th 
century, physics and chemistry were united by quantum</span><span style="font-family: Times">
</span><span lang="RU" style="font-family: Times">theory, which explored the 
fine structure of matter and its associated energy. At that time, biology was a 
field concerned with the direct observation and description of complex natural 
phenomena. Then, beginning in the 1920s and 30s, ideas derived from quantum 
mechanics began</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">to infuse biology. It began to be 
understood that the behavior of any complex biological system,</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">whether it is a bacterium, a 
plant, a frog, a leopard, or a human being, is based on the physical</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">and chemical behavior of 
elementary particles.</span><span style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">Molecular biology borrowed from the 
physical sciences the rigorous use of model </span>
<span lang="RU" style="font-family: Times">systems—simplified abstractions of 
reality in which variables could be limited and experimental</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">situations could be 
controlled. The development of molecular biology was in large part driven</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">by the quest to find 
increasingly purer and more powerful abstractions of essential biological</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">processes. Experimental 
systems using complex, multicellular organisms (such as pea plants and</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">fruit flies used in the early 
1900s) began to be replaced first by simple, one-cell organisms (bacteria and 
their attendant viruses, beginning in the 1940s) and then by purified cellular 
components (such as DNA and proteins, beginning in the 1960s).</span></p>
<p class="MsoNormal"><b><span lang="RU" style="font-family: Times">II. Tools</span></b></p>
<p class="MsoNormal"><i><span lang="RU" style="font-family: Times">A. 
Restriction Enzymes</span></i></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">Restriction 
enzymes were first discovered in the 1950s when evidence for a sort of primi-</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">tive immune 
system was found to exist in bacteria. It was observed that certain strains of&nbsp;<i>E. 
Coli</i></span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">were resistant to infection by 
various bacteriophages (viruses that infect bacteria). This phenomenon seemed to 
be a property of the bacterial cell itself, an ability to restrict the growth 
and</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">replication of the attacking phages. 
In 1962, at the University of Geneva, it was found that the</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">resistant bacterium possessed 
an enzyme system that selectively recognized and destroyed foreign phage DNA 
within the bacterial membrane and at the same time modified the chromosomal DNA 
of the bacterium to prevent self-destruction.</span><span style="font-family: Times">&nbsp;
</span><span lang="RU" style="font-family: Times">Several years later, at 
Harvard University, extracts of&nbsp;<i>E. coli&nbsp;</i>were made which were</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">found to efficiently cleave 
phage DNA. These extracts contained the first known restriction</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">endonucleases, enzymes that 
attack and digest internal regions of the DNA of an invading bacteriophage but 
not that of the host. This was shown to occur because, in addition to a 
restriction (cutting) activity, these enzymes also possessed a modification 
(protecting) activity. However,</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">because the cutting activity 
of these enzymes was not specific, i.e., the enzymes cut at random</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">places within the phage DNA, 
they were of no practical value as tools for manipulating DNA.</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">Then, in 1970, at Johns 
Hopkins University, a new kind of restriction enzyme was</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">found, one that&nbsp;<i>could&nbsp;</i>cut 
at a specific site within the phage DNA, while at the same time protecting the 
same site on the bacterial host’s DNA. The protection was accomplished by adding 
a</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">methyl group to the host’s site, thus 
disguising it. This new enzyme was isolated from</span><span lang="RU" style="font-family: Times">
</span><i><span lang="RU" style="font-family: Times">Haemophilus influenzae</span></i><span lang="RU" style="font-family: Times">, 
and was called HindII. It cleaved DNA predictably, cutting within a specific</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">sequence of base pairs, 
regardless of whether the DNA was from a small virus that infects</span><span style="font-family: Times">
</span><span lang="RU" style="font-family: Times">monkeys (Simian virus 40 
(SV40)) or a virus that infects&nbsp;<i>E. coli</i>. The resulting restriction 
fragments could then be separated by size in an electrical field. The order of 
the fragments (and</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">corresponding restriction sites) 
could be deduced in the 5,000 nucleotide circular chromosome</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">to create a restriction enzyme 
map that was then related to the existing genetic map of SV40.</span></p>
<p class="MsoNormal"><i><span lang="RU" style="font-family: Times">B. Agarose 
Gel Electrophoresis</span></i></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">Gel 
electrophoresis takes advantage of the fact that, as an organic acid, DNA can be</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">negatively 
charged, depending on the alkalinity of the surrounding buffer. DNA owes its 
acidity</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">to phosphate groups that alternate 
with deoxyribose to form the rails of the double helix ladder.</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">In a slightly basic solution, 
negatively charged oxygens radiate from phosphates on the outside</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">of the DNA molecules. When 
placed in an electric field, DNA molecules are attracted toward</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">the positive pole (anode) and 
repelled from the negative pole (cathode).</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">During electrophoresis, DNA 
fragments sort by size in the gel. The porous gel matrix</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">acts as a molecular sieve 
through which smaller molecules can move more easily than large</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">ones, and the distance moved 
by a DNA fragment is inversely proportional to its molecular</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">weight. In a given period of 
time, smaller restriction fragments migrate relatively far from the</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">origin compared with larger 
fragments. DNA fragments in different size ranges can be separated</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">by adjusting the agarose 
concentration: low agarose concentrations produce a loose gel that separates 
large fragments, whereas a high concentration produces a stiff gel that resolves 
small fragments. In the gel we use in this experiment, we chose an agarose 
matrix that can efficiently separate DNA fragments ranging in size from about 
100 nucleotides to about 5,000 nucleotides. To</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">do this, we use a 1</span><span style="font-family: Times">.2
</span><span lang="RU" style="font-family: Times">% gel, which turns out 
experimentally to be the size of molecular sieve that</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">will effectively sort, and 
resolve, the fragments generated by our digestions.</span></p>
<p class="MsoNormal"><b><span lang="RU" style="font-family: Times">III. 
Experimental Objective</span></b></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">This experiment 
involves mapping the number and relative positions of cutting sites for</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">three 
restriction enzymes, EcoR1, HincII, and PvuII, on circular plasmid pBR322, the 
first artificially produced plasmid to be used routinely in recombinant DNA 
experiments. This plasmid</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">was originally derived from 
the bacteria&nbsp;<i>E. coli&nbsp;</i>and was constructed to have two antibiotic 
resistance genes, tetracycline and ampicillin. Restriction enzyme mapping is 
important because it is</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">the first step in characterizing a 
DNA fragment that has been ligated into a vector (usually a</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">plasmid), providing necessary 
data for applications such as DNA sequencing, gene localization,</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">and site-directed mutagenesis.</span></p>
<p class="MsoNormal"><b><span lang="RU" style="font-family: Times">IV. Notes on 
Experimental Procedure</span></b></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">Since molecular 
biology deals with small quantities, be prepared to measure liquid quantities on 
the order of microliters (a millionth of a liter) (in general, wear gloves for 
work in</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">molecular biology). A few notes on 
the use of the micropipettor may be helpful:</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">1. Rotate the 
volume adjuster to the desired setting, which is shown in a window on the side 
of</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">the micropipettor. The line between 
two of the digits corresponds to a decimal point.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">2. Place the 
appropriate tip on the end of the micropipettor. The p10s (those pipettes that 
can</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">pull up and expell a total volume of 
10 &#956;ls use a very small diameter tip compared with a p20</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">or above.&nbsp;<i>Use 
a new tip for each new solution</i>.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">3. When 
withdrawing or expelling fluid from a microfuge tube, always hold the tube 
firmly</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">between thumb and forefinger. Hold 
the tube at nearly eye level to observe the change in the</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">fluid level in the pipette 
tip.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">4. Each tube 
must be held in the hand during each manipulation. Grasping the tube body,</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">rather than the 
lid, provides more control and avoids contamination.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">5. Hold the 
pipettor almost vertical when filling.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">6. Most 
micropipettors have a two-position plunger with “friction stops.” Depressing to 
the first</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">stop selects the desired volume to be 
withdrawn from the tube. Depressing to the second stop</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">introduces an additional 
volume of air to blow out any solution remaining in the tip. Pay attention to 
these friction stops, which can be felt with the thumb.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">7. To withdraw a 
sample from the reagent tube:</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">a. Depress the 
plunger to the first stop and hold it in this position while you dip the tip</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">into the 
solution to be pipetted and gradually release the plunger. You should see liquid 
being</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">drawn into the tip.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">b. Slide the 
pipette tip out along the inside wall of the reagent tube to dislodge excess</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">droplets that 
may be adhering to the outside of the tip.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">c. Check that 
there is no air space at the very end of the tip. To avoid pipetting errors,</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">learn to 
recognize the approximate level that particular volumes fill the pipette tip.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">8. To expel a 
sample into the reaction tube:</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">a. Put the 
pipette tip into the reaction tube into which you wish to empty the sample,</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">near the bottom, 
touching the tip to the inside wall of the tube. This will allow a capillary 
effect</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">to help draw the 
fluid out of the tip.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">b. Slowly 
depress the plunger to the first stop to expel the sample. Now depress the</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">plunger to the 
second stop to blow out the last bit of fluid.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">c. Holding the 
plunger in the depressed position, slide the pipette out of the reaction</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">tube to avoid 
sucking any liquid back into the tip.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">d. Eject the tip 
into a waste beaker near you on the lab bench.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">9. To prevent 
cross contamination of reagents:</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">a. Add the 
appropriate amounts of a single reagent sequentially to all reaction tubes.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">10. Then, using 
a new tip, add the next reagent, and so on.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">b. Release each 
reagent drop onto a new location on the inside wall, near the bottom of</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">the reaction 
tube, being careful not to touch any drops already added to the tube. The same 
tip</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">can be used to 
pipette the same reagent into each reaction tube.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">c. Use a fresh 
tip for each new reagent to be pipetted.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">d. If a tip 
becomes contaminated, switch to a new one.</span></p>
<p class="MsoNormal"><b><span lang="RU" style="font-family: Times">V. Mapping 
pBR322</span></b></p>
<p class="MsoNormal"><i><span lang="RU" style="font-family: Times">A. Setting Up 
the Restriction Digests</span></i></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">1. Label </span>
<span style="font-family: Times">six</span><span lang="RU" style="font-family: Times"> 
1.5-ml </span><span style="font-family: Times">microcentrifuge</span><span lang="RU" style="font-family: Times"> 
tubes, as shown in the matrix below. The restriction enzyme</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">digests will take place in 
these tubes.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">2. Use the 
matrix below as a checklist while adding the reagents to each reaction tube. 
Read</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">down each column, adding the same 
reagent to each appropriate tube (all these reagents are in</span><span style="font-family: Times"> 
either in </span><span lang="RU" style="font-family: Times">the ice buckets on 
your bench</span><span style="font-family: Times"> or in stock tubes at the 
front of the lab</span><span lang="RU" style="font-family: Times">). Add the 
reagents in the order described in the steps that follow</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">the matrix.</span></p>
<p class="MsoNormal"><span lang="RU">&nbsp; <br>
</span>Student Set-ups:</p>
<p class="MsoNormal"><u><span lang="RU">Digest Setups</span></u><span lang="RU">
</span></p>
<table class="MsoNormalTable" border="1" cellspacing="0" cellpadding="0" style="border-collapse: collapse; border: medium none">
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">Tube # </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">pBR322-DNA </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Buffer # and amount (10X) </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Enzyme </span></p>
    <p class="MsoNormal"><span lang="RU">(1 µL of each enzyme) </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">BSA&nbsp; </span></p>
    <p class="MsoNormal"><span lang="RU">(10X)&nbsp; </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">dH</span><sub><span lang="RU" style="font-size: 7.5pt">2</span></sub><span lang="RU">O
    </span></td>
  </tr>
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">1 </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">3 µL </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">EcoRI Buffer/ 1 µL </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">EcoRI </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;1 µL </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;4 µL </span></td>
  </tr>
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">2 </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">3 µL </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Buffer 3/ 1 µL </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">HincII </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;1 µL </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;4 µL </span></td>
  </tr>
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">3 </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">3 µL </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Buffer 3/ 1 µL </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">PvuII </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;1 µL </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;4 µL </span></td>
  </tr>
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">4. </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">3 µL </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Buffer 3/ 1 µL </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">PvuII, HincII </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;1 µL </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;3 µL </span></td>
  </tr>
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">5. </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">3 µL </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Buffer 3/ 1 µL </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">BamHI, PvuII </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;1 µL </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;3 µL </span></td>
  </tr>
  <tr>
    <td width="55" valign="top" style="width: 41.25pt; border: medium none; padding: 0in">
    <p class="MsoNormal"><span lang="RU">6. </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">3 µL </span></td>
    <td width="137" valign="top" style="width: 102.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">Buffer 3/ 1 µL </span></td>
    <td width="186" valign="top" style="width: 139.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">EcoRI, HincII, PvuII </span></td>
    <td width="66" valign="top" style="width: 49.5pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;1 µL </span></td>
    <td width="73" valign="top" style="width: 54.75pt; border-color: inherit; padding-left: 4; padding-right: 4; padding-top: 1; padding-bottom: 1" bordercolor="#C0C0C0">
    <p class="MsoNormal"><span lang="RU">&nbsp;2 µL </span></td>
  </tr>
</table>
<p class="MsoNormal"><span lang="RU">&nbsp;</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">3. Add 3 &#956;l of 
pBR322 to the reaction tubes</span><span style="font-family: Times">.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">4. Use a fresh 
tip to add </span><span style="font-family: Times">BSA (bovine serum albumin- 
helps the enzymes work better).</span><span lang="RU" style="font-family: Times">.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">5. Use a fresh 
tip to add the appropriate volume of distilled water to each reaction tube (the</span><span style="font-family: Times">&nbsp;
</span><span lang="RU" style="font-family: Times">amount needed will bring the 
total volume of the reaction to 10 &#956;l</span><span lang="RU" style="font-size: 10.0pt; font-family: Times">.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">6. Use a fresh 
tip to add 1 &#956;l of 10X buffer </span><span style="font-family: Times">(only the 
EcoRI uses a different buffer)</span><span lang="RU" style="font-family: Times"> 
to each reaction tube.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">7. Use a fresh 
tip to add each different restriction enzyme to the appropriate reaction tube.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">8. After 
carefully adding the required reagents to each tube, close the tops of the tubes 
and</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">microfuge them briefly in the 
microfuge on your bench in order to bring all the reagents to the</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">bottom of the tube. (The 
reagents need to be near each other in order to react.)</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">9. Place all of 
the reaction tubes in a “floater” and place the floater in the 37</span><sup><span lang="RU" style="font-size: 8.5pt; font-family: Times">o</span></sup><span lang="RU" style="font-family: Times">C 
water bath. Incubate for </span><span style="font-family: Times">2 hours.</span><span lang="RU" style="font-family: Times">.</span></p>
<p class="MsoNormal"><i><span lang="RU" style="font-family: Times">B. Casting a 
1</span><span style="font-family: Times">.2</span><span lang="RU" style="font-family: Times">% 
Agarose Gel</span></i></p>
<p class="MsoNormal"><span style="font-family: Times">1</span><span lang="RU" style="font-family: Times">. 
Let the agarose cool to a temperature that will allow you to touch the flask 
(this will be about</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">60°C)</span><span style="font-family: Times">.
</span><span lang="RU" style="font-family: Times">While the agarose is cooling, 
make your gel mold liquid-tight by </span><span style="font-family: Times">
putting masking tape</span><span lang="RU" style="font-family: Times"> on either 
end are pressed against the sides of the box.</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">Insert the thick side of the 
comb into the slot at the top of the mold. You will be creating</span><span style="font-family: Times"> 
8</span><span lang="RU" style="font-family: Times"> wells.</span></p>
<p class="MsoNormal"><span style="font-family: Times">2</span><span lang="RU" style="font-family: Times">. 
Pour your gel into the gel mold and let it set. If you see any bubbles on the 
surface of the gel</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">after pouring, scoot them to the 
edges with a pipette tip.</span></p>
<p class="MsoNormal"><span lang="RU">Incubate for ~2 hours @ 37C. &nbsp;Then 
refrigerate or freeze until ready to load gel. </span></p>
<p class="MsoNormal"><span style="font-family: Times">&nbsp;</span></p>
<p class="MsoNormal"><span style="font-family: Times">Next Day:</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">6. </span>
<span style="font-family: Times">Remove your gel from the baggie</span><span lang="RU" style="font-family: Times">.</span><span style="font-family: Times">&nbsp; 
Carefully remove the making tape.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">7. Pour running 
buffer (on your benches in large bottles) into the gel box so that it covers the 
gel</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">by about a</span><span style="font-family: Times"> 
few millimeters (make sure the well edges are submerged)</span><span lang="RU" style="font-family: Times">.</span></p>
<p class="MsoNormal"><i><span lang="RU" style="font-family: Times">C. Load Gel 
and Electrophorese</span></i></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">1. </span>
<span style="font-family: Times">Retrieve your tubes from the front table</span><span lang="RU" style="font-family: Times">.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">2. Add 2</span><span style="font-family: Times">.5</span><span lang="RU" style="font-family: Times"> 
&#956;l of loading dye to each reaction tube. The loading dye is a dense solution 
containingvarious pigments. The solvent here is a dense sucrose solution that 
should allow the colored</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">solution to sink into the 
wells. The blue and purple pigments will tell us when to stop the current since 
one of the pigments travels at about the same rate as the smallest fragment of 
the</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">DNA does.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">3. After adding 
the dye, close the tops of the tubes, and microfuge.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">4.. . </span>
<span style="font-family: Times">Load</span><span lang="RU" style="font-family: Times"> 
12</span><span style="font-family: Times">.5</span><span lang="RU" style="font-family: Times"> 
&#956;l from each reaction tube (the total volume in the tube) into a separate well 
in</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">the gel, beginning with the second 
well. Note that you will now need the p20 micropipette in</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">order to accommodate the total 
volume of 12</span><span style="font-family: Times">.5</span><span lang="RU" style="font-family: Times"> 
&#956;l. Make a note for yourself about which digest is</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">in which well. </span>
<span style="font-family: Times">L</span><span lang="RU" style="font-family: Times">oad 
from the left, proceeding in order through the last digest from the tube </span>
<span style="font-family: Times">(see chart below)</span><span lang="RU" style="font-family: Times">.&nbsp;</span><b><i><span lang="RU" style="font-size: 10.5pt; font-family: Times">Note</span></i></b><span lang="RU" style="font-size: 10.5pt; font-family: Times">:&nbsp;<b>Use 
a new</b></span><span style="font-family: Times">&nbsp; </span><b>
<span lang="RU" style="font-size: 10.5pt; font-family: Times">tip for each 
sample.&nbsp;</span></b><span lang="RU" style="font-family: Times">Be careful not to 
push the tip of the pipette through the bottom of the</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">gel.</span></p>
<p class="MsoNormal"><span style="font-family: Times">5</span><span lang="RU" style="font-family: Times">. 
Connect the leads to the power source and turn it on. Red indicates the positive 
end and</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">black indicates the negative. 
Remember that since DNA is negative with the buffer we are using,</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">you will want to position the 
leads so that the DNA is running toward the positive end.</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">7. 
Electrophorese until the leading dye has run down the gel to between the numbers 
4 and 5</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">which appear on the side of the gel. 
Then turn the current off, or if another gel is still running</span><span lang="RU" style="font-family: Times">
</span><span lang="RU" style="font-family: Times">on your power source, 
disconnect the electrodes for your gel.</span><span style="font-family: Times"> 
(200V for 20 minutes).</span></p>
<p class="MsoNormal"><i><span lang="RU" style="font-family: Times">D. View and 
Photograph</span></i></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">1. Wearing 
gloves, take out the gel mold with the gel in it and slide the gel into </span>
<span style="font-family: Times">staining tray o</span><span lang="RU" style="font-family: Times">nwhich 
you have written </span><span style="font-family: Times">the table and period 
number</span><span lang="RU" style="font-family: Times"> (on the edge of </span>
<span style="font-family: Times">the staining tray).</span></p>
<p class="MsoNormal"><span style="font-family: Times">2. Pour enough Fast-Blast 
Stain (1X) to cover the gel and put on the rotating table.</span></p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal"><span lang="RU">Loading Pattern for Gel </span></p>
<table class="MsoNormalTable" border="1" cellspacing="0" cellpadding="0" width="657" style="width: 492.75pt; border-collapse: collapse">
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal">
    <span lang="RU" style="font-family: Verdana; color: black">Use 2.5 µl of 6X 
    Blue Gel Loading Dye for the 10 µl reaction. Mix well before loading gel. 
    &nbsp;Load all the sample on the gel</span><span lang="RU"> </span></p>
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal">Lane contents<span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Band sizes (bp)<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 1<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">HindIII Digest Control<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 2<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">EcoRI<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 3<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">HincII<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 4<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">PvuII<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 5<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">PvuII, HincII<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 6<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">BamHI, PvuII<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Lane 7<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">EcoRI, HincII, PvuII<br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU"><br>
    <![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></td>
  </tr>
  <tr>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">Empty</span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">&nbsp;</span></td>
    <td width="33%" style="width: 33.32%; padding: 2.25pt">
    <p class="MsoNormal"><span lang="RU">&nbsp;</span></td>
  </tr>
</table>
<p class="MsoNormal"><span lang="RU"><br>
<![if !supportLineBreakNewLine]><br>
&nbsp;<![endif]></span></p>
<p class="MsoNormal"><span lang="RU">Bio-Rad: 0.25X TAE; 200 V, 20 minutes<br>
Stain overnight in FastBlast Blue stain 1X</span></p>
<p class="MsoNormal"><span lang="RU"><br>
</span>Next Day:</p>
<ol style="margin-top: 0in; margin-bottom: 0in" start="1" type="1">
  <li class="MsoNormal">View gel, measure distances of bands from the wells, 
  record in data table.</li>
  <li class="MsoNormal">HindIII (known base pair sizes starting from the well; 
  23,130, 9416, 6557, 4361, 2322, 2027, 564.</li>
  <li class="MsoNormal">Use a semi-log graph to generate a standard curve for 
  HindIII control (x-axis- distance band migrated (mm); y-axis (semi-log from 
  100 to 100,000).</li>
  <li class="MsoNormal">Determine sizes of unknown bands.</li>
  <li class="MsoNormal"><font FACE="TimesNewRomanPSMT">
  <p ALIGN="LEFT">1. Draw a simple pBR322 restriction map showing the sites for 
  enzymes used, in relative locations;</p>
  <p ALIGN="LEFT">2. Predict the expected number of linear DNA bands following 
  complete digestion with each enzyme;</p>
  <p ALIGN="LEFT">3. Predict the expected sizes of those bands and use the size 
  marker to help identify them on your gel;</p>
  <p ALIGN="LEFT">4. Explain what you see when pBR322 is digested with HincII;</p>
  <p ALIGN="LEFT">5. Explain why HincII&nbsp; is expected to have many more 
  restriction sites in pBR322 than the otherenzymes;</p>
  <p>6. Comment on any unexpected bands in your gel.</p>
  <p>(questions from
  <a href="http://www.csus.edu/indiv/r/rogersa/protpBR322.pdf">
  http://www.csus.edu/indiv/r/rogersa/protpBR322.pdf</a> )</p>
  </font>
  <p>&nbsp;</li>
</ol>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal">&nbsp;</p>
<p class="MsoNormal"><b>
<span lang="RU" style="font-size: 14.5pt; font-family: Times">DNA MAPPING: 
Post-Lab Questions</span><span style="font-size: 14.5pt; font-family: Times"> to 
consider for your discussion.</span></b></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">1. What are two 
functions of the loading dye?</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">2. How does
</span><span style="font-family: Times">Fast-Blast visible dye</span><span lang="RU" style="font-family: Times"> 
stain DNA?</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">3. To cast the 
electrophoresis gels, why did we use buffer to dissolve the agarose; why not 
just</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">use water? Please suggest 2 reasons.</span></p>
<p class="MsoNormal"><span style="font-family: Times">4</span><span lang="RU" style="font-family: Times">. 
To run our digests in the electrophoresis chamber, why did we use a buffer? Why 
not just use</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">water?</span></p>
<p class="MsoNormal"><span style="font-family: Times">5</span><span lang="RU" style="font-family: Times">. 
In order to run our electrophoresis chambers, why did we cover the gels with 
just a little</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">buffer; why not use quite a lot?</span></p>
<p class="MsoNormal"><span style="font-family: Times">6</span><span lang="RU" style="font-family: Times">. 
What would happen if we ran the gel with the electrodes reversed?</span></p>
<p class="MsoNormal"><span style="font-family: Times">7</span><span lang="RU" style="font-family: Times">. 
What might account for “smearing” (i.e., no discrete bands, just one big blurry 
smear)?</span></p>
<p class="MsoNormal"><span style="font-family: Times">8</span><span lang="RU" style="font-family: Times">. 
Why might various faint bands appear in some of the lanes?</span></p>
<p class="MsoNormal"><span style="font-family: Times">9</span><span lang="RU" style="font-family: Times">. 
How would decide whether these faint bands were “real” bands or just artifacts?</span></p>
<p class="MsoNormal"><span lang="RU" style="font-family: Times">1</span><span style="font-family: Times">0</span><span lang="RU" style="font-family: Times">. 
Restriction enzymes are generally named for the organism from which they were 
derived.</span><span lang="RU" style="font-family: Times"> </span>
<span lang="RU" style="font-family: Times">From what organism was PvuII derived?</span></p>
<p class="MsoNormal"><span style="font-family: Times">&nbsp;</span></p>
<p class="MsoNormal"><span style="font-family: Times">Rubric:&nbsp; (23)</span></p>
<p class="MsoNormal"><span style="font-family: Times">Title (1)</span></p>
<p class="MsoNormal"><span style="font-family: Times">Data Table (4)</span></p>
<p class="MsoNormal"><span style="font-family: Times">HindIII Standard graph 
with notations to show how you determined unknown band sizes in basepairs. (4) 
(some helpful info...for a resource, don’t email the other teacher whose site 
I’ve linked &nbsp;<a href="http://www.occc.edu/bbdiscovery/documents/MWfromgel.htm" style="color: blue; text-decoration: underline; text-underline: single">http://www.occc.edu/bbdiscovery/documents/MWfromgel.htm</a> 
) </span></p>
<p class="MsoNormal"><span style="font-family: Times">&nbsp;</span></p>
<p class="MsoNormal"><span style="font-family: Times">Possible Restriction Maps 
(at least two) to show possible order of fragments on pBR322 BASED ON YOUR DATA 
ONLY. (8)&nbsp; </span></p>
<p class="MsoNormal"><span style="font-family: Times">Discussion (background 
info and compare to the actual <a href="pBR322_map.pdf">pBR322 plasmid map</a>(4)</span></p>
<p class="MsoNormal"><span style="font-family: Times">References (2)</span></p>

</body>

</html>
