<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html><head><meta content="text/html; charset=ISO-8859-1" http-equiv="content-type"><title>WhalesActivityv.2.htm</title></head><body>
<p style="margin-bottom: 0.1in; line-height: 0.18in;" align="center" lang="en-US">
<span id="Frame1" style="border: medium none ; padding: 0in; background: rgb(255, 255, 255) none repeat scroll 0%; position: absolute; top: 1.57in; left: 4.89in; width: 3.16in; height: 0.02in; -moz-background-clip: initial; -moz-background-origin: initial; -moz-background-inline-policy: initial;" dir="ltr">
	<p style="margin-bottom: 0.65in;" align="left" lang="en-US"></p>
	<p style="margin-bottom: 0.65in;" align="left" lang="en-US"><br><br>
	</p>
</span><font color="#000000"> </font></p><div style="text-align: left;"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font color="#000000"><font size="2"><b>&nbsp;WALRUSES AND WHALES AND SEALS, OH MY! (Adapted from Kim Foglia&nbsp;2010) </b></font></font></font></font></font></div>
<p style="margin-right: 2.65in; line-height: 0.18in;" align="left" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">Walruses
and whales are both marine mammals. So are dolphins, seals, and
manatee. They all have streamlined bodies, legs reduced to flippers,
blubber under the skin and other adaptations for survival in the
water. Although mammals evolved on land, these species have returned
to the sea. Did they evolve from a single ancestor who returned to
the ocean, or were there different return events and parallel
evolution? We can&#8217;t go back in time to observe what happened, but
DNA sequences contain evidence about the relationships of living
creatures. From these relationships, we can learn about the
evolutionary history of marine mammals. </font></font></font>
</p>
<p style="margin-right: 2.65in; line-height: 0.18in;" align="left" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">In
this lab, we will use sequence information in GenBank (the public
repository of all known DNA sequences from many species) and
bioinformatics software to test hypotheses about the relationship
between aquatic mammals (seals, whales, dolphins, walruses, manatees,
and sea otters) and their potential ancestral relationship to land
mammals. </font></font></font>
</p>
<p style="line-height: 0.18in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">We
will use a protein that all mammals share: the hemoglobin beta
protein. Hemoglobin is a good test molecule since it shows both
conservation across species (since it performs the essential function
of carrying oxygen in the blood), and variation between species.
Species with unique challenges such as holding their breath for long
underwater dives, may have evolved changes in their hemoglobin which
improved their supply of oxygen. In addition, hemoglobin has been
studied by many evolutionary biologists, so sequences are available
in GenBank from many different organisms. </font></font></font>
</p>
<p align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 11pt;" size="3"><b>PROCEDURE
</b></font></font></font>
</p>
<p style="line-height: 0.18in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">In
this lab, we will be testing hypotheses about the evolutionary
ancestry of different marine mammals. To repeat, we are trying to
answer the question: Did marine mammals evolve from a single ancestor
who returned to the ocean, or were there distinct return events from
separate ancestors? As a starting point let&#8217;s hypothesize that
marine mammals have a single common land mammal ancestor. </font></font></font></font>
</p>
<p style="line-height: 0.18in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><u><b>Part
A</b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">: First, we
will explore the relationship of the marine mammals to each other vs.
their evolutionary relationship to land mammals. To do this, we will
test whether seals and whales are more closely related to each other
than either of them are to representative land mammals: dogs (land
carnivores) or cows (land herbivores). This exercise will mainly
train you in using the bioinformatics software. </font></font></font></font>
</p>
<p style="margin-bottom: 0in; line-height: 0.18in;" align="left" lang="en-US">
<font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><u><b>Part
B</b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">: Second,
each student will then develop a cladogram which includes a selection
of marine mammals and land mammals which represent the major
mammalian orders. You will then use this phylogenetic tree to test
our hypothesis that all marine mammals have a single common land
mammal ancestor. </font></font></font></font>
</p>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>
<p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">To
proceed through this exercise, first visit the Next Generation
Biology workbench (</font></font><a href="http://www.ngbw.org/"><font color="#0000ff"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><u>http://www.ngbw.org</u></font></font></font></a><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">),
and create an account. To do this, click on the &#8220;Use the Workbench
Now&#8221; button. A new page will open. On that page, click on the
button that says &#8220;Create an account.&#8221; This will open a form int
whoch you enter your personal information. </font></font><font face="Arial, sans-serif"><font size="2">If
you need help, paste this address into your browser:
</font></font><a href="http://www.ngbw.org/help/register.htm"><font color="#0000ff"><font face="Arial, sans-serif"><font size="2"><u>http://www.ngbw.org/help/register.htm</u></font></font></font></a><font face="Arial, sans-serif"><font size="2">,
or just click the help tab on the upper right hand side of the site.
Once your registration is complete, you will be logged in to your
personal area. </font></font></font></font></font>
</p>
<p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">The
NGBW allows you to store data and tasks in folders just like MS
Outlook or other mail clients. So before working here, you must
create at least one folder. Click on the button that says &#8220;Create
New Folder&#8221;. Name your folder, and save the name. The folder you
created will appear on the left side of the screen. If you get stuck,
here is the help link (</font></font><a href="http://www.ngbw.org/help/create_folder.htm"><font color="#0000ff"><font face="Arial, sans-serif"><font size="2"><u>http://www.ngbw.org/help/create_folder.htm</u></font></font></font></a><font face="Arial, sans-serif"><font size="2">).
 Open a second browser </font></font><font face="Arial, sans-serif"><font size="2">page
or new tab so you can toggle between the NGBW and the sequence
database we use. Now,</font></font><font face="Arial, sans-serif"><font size="2">
we can begin the exercise</font></font><font face="Arial, sans-serif"><font size="2">.</font></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">
</font></font></font></font></font>
</p>
<p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><b>PART
A </b></font></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><b><br></b></font></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><b>FINDING
AMINO ACID SEQUENCES  </b></font></font></font></font></font>
</p>
<ol><li><p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">.First
	we need to get the sequence data for the hemoglobin protein from
	marine and land animals: seals, whales, dogs, cows. Several sites
	offer this kind of data. For example, </font></font><font style="font-size: 10pt;" size="2"><u><b>UniProt</b></u></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">,
	is a highly curated protein sequence database hosted by the EBI
	(</font></font><a href="http://www.uniprot.org/"><font color="#0000ff"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><u>http://www.uniprot.org/</u></font></font></font></a><font style="font-size: 10pt;" size="2"><u><b>)
	</b></u></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">(Note:
	You can also go to Google and search for &#8220;UNIPROT&#8221; and it will
	always come up as the first link). In this case we will use UNIPROT
	to retrieve sequences, and a web site called the Next Generation
	Biology Workbench (http://www.ngbw.org), which provides access to
	both the sequences from international data repositories and to the
	tools we require today to analyse that data.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">.
	At the top of the UniProt home page, there is a search tool. Make
	sure the drop down under </font><font face="NFKRIX+ArialMT, Arial, sans-serif"><i><b>Search
	in</b></i></font><font face="NFKRIX+ArialMT, Arial, sans-serif">
	says &#8220;Protein Knowledgebase: UniProtKB. You will search for you
	data here. Just as with any Google search, effective searching for
	data in a protein database depends on choosing effective search
	terms. One thing you need to know: the hemoglobin beta gene and
	protein is known as &#8220;</font><u><b>HBB</b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;
	in UniProt and other databases. However, there are many</font><font face="NFKRIX+ArialMT, Arial, sans-serif">,</font><font face="NFKRIX+ArialMT, Arial, sans-serif">
	many hemoglobin sequences available, we need to be as specific as
	possible about the identity of the sequences we want. That way we
	won&#8217;t be overwhelmed by search results we are not interested in.
	For now, we want to gather sequences from the harbor seal, the minke
	whale (a baleen whale), </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Canis
	familiaris </i></font><font face="NFKRIX+ArialMT, Arial, sans-serif">(dog),
	</font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Bos
	taurus </i></font><font face="NFKRIX+ArialMT, Arial, sans-serif">(cow),
	and red kangaroo. To retrieve these sequences, type in the query
	box: HBB and the name of the organism you want. For example, type
	&#8220;HBB minke whale&#8221; and click &#8220;Search&#8221;.</font></font></font></font></p>
</li></ol>

<p style="margin-left: 0.25in; margin-bottom: 0.25in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font color="#000000"><u><b>NOTE</b></u></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">:
This database is queried by computers humans, so it does not always
return results that seem organized logically. Don&#8217;t get frustrated;
be patient. Your search will often return several answers; but the
highly curated UniProt DB will help minimize this issue. Look at the
results for a record name that contains the terms Hemoglobin, and
Beta. That will be the correct protein! You can insure this by
including these as search terms. </font></font><font face="NFKRIX+ArialMT, Arial, sans-serif">Because
short strings like &#8220;cow&#8221; and &#8220;dog&#8221; are fairly non-specific
and occur frequently in databases, you should use the scientific
names</font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>
Bos taurus </i></font><font face="NFKRIX+ArialMT, Arial, sans-serif">and
</font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Canis
familiaris </i></font><font face="NFKRIX+ArialMT, Arial, sans-serif">as
search terms.</font></font></font></font></p>
<ol start="3"><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">.
	Find the sequence you need among the returned sequences. Now save it
	for further analysis using the NGBW. To do this, click the Accession
	number hyperlink of the sequence you want. A page will open with a
	wide variety of information about the sequence. Today, we just want
	to recover the protein sequence alone. To do this, click the gold
	FASTA link at the extreme right of the sequence page (its small, but
	it&#8217;s there!). This opens a file containing the amino acid sequence
	of the protein, represented using single letters for each of the 20
	naturally occurring amino acids (A=alanine, M=methionine, P=proline,
	etc). Here is how the file looks:</font></font></font></p>
</li></ol>

<p style="margin-bottom: 0in; line-height: 100%;"><font face="Courier New, monospace"><font size="2">&gt;sp|P02107|HBB_MACRU
Hemoglobin subunit beta OS=Macropus rufus GN=HBB PE=1 SV=1</font></font></p>
<p style="margin-bottom: 0in; line-height: 100%;"><font face="Courier New, monospace"><font size="2">VHLTAEEKNAITSLWGKVAIEQTGGEALGRLLIVYPWTSRFFDHFGDLSNAKAVMGNPKV</font></font></p>
<p style="margin-bottom: 0in; line-height: 100%;"><font face="Courier New, monospace"><font size="2">LAHGAKVLVAFGDAIKNLDNLKGTFAKLSELHCDKLHVDPENFKLLGNIIVICLAEHFGK</font></font></p>
<p style="margin-bottom: 0in; line-height: 100%;"><font face="Courier New, monospace"><font size="2">EFTIDTQVAWQKLVAGVANALAHKYH</font></font></p>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><br>While
this file isn&#8217;t nice for humans to read, the workbench knows how to
access and utilize this sequence information for your analysis. The
first line is called a &#8220;header&#8221;. It contains information
describing the sequence. The reminder represents the sequence of
amino acids in the protein. Right click on the sequence, and &#8220;copy&#8221;
it.</font></font></font></p>

<ol start="4"><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">.
	Transfer the sequence to the NGBW by going to the tab or browser
	page with your open NGBW account, and click the &#8220;Data&#8221; folder.
	This opens a data management area. In the &#8220;Label&#8221; entry box,
	give the sequence a name you will recognize easily (for example, the
	common name of the organism). </font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">In
	the &#8220;</font></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><i><b>Add
	Data</b></i></font></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;
	area, click &#8220;</font></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><i><b>Upload
	Files</b></i></font></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;
	Use the right click, and paste the data into the form box labeled
	&#8220;</font></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><i><b>Or,
	enter your data:</b></i></font></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;
	Paste your data into this block. Before saving it, let&#8217;s change
	the </font></font><font color="#000000"><u><b>sequence header </b></u></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">at
	the top of the protein sequence that we pasted into the box. This
	will help us later. The header is changed to be more informative
	when we make a tree later. This header can be edited for clarity,
	</font></font><font color="#000000"><u><b>but you MUST preserve the
	&#8220;&gt;&#8221; symbol</b></u></font><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif">.
	You can use the scientific name or the common name to identify your
	sequence. </font></font></font></font></font>
	</p>
</li></ol>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>
<p style="margin-left: 0.25in; margin-bottom: 0.16in; line-height: 0.18in;" align="left" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">For
example: The red kangaroo sequence begins like this: </font></font></font>
</p>
<p style="margin-left: 0.25in; margin-bottom: 0.16in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="Courier New, monospace"><font size="2">&gt;sp|P02107|HBB_MACRU
Hemoglobin subunit beta OS=Macropus rufus GN=HBB PE=1 SV=1</font></font></font></p>
<p style="margin-left: 0.25in; margin-bottom: 0.16in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">This
can be edited to simply say this: </font></font></font></font></font>
</p>
<p style="margin-left: 0.25in; margin-bottom: 0in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">&gt;Red_kangaroo</font></font></font></p>
<p style="text-indent: 0.25in; margin-bottom: 0in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">VHLTAEEKNAITSLWGKVAIEQTGGEALGRLLIVYPWTSRFFDHFGDLSNAKAVMGNPKV</font></font></font></p>
<p style="margin-left: 0.25in; margin-bottom: 0in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">LAHGAKVLVAFGDAIKNLDNLKGTFAKLSELHCDKLHVDPENFKLLGNIIVICLAEHFGK</font></font></font></p>
<p style="margin-left: 0.25in; margin-bottom: 0in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">EFTIDTQVAWQKLVAGVANALAHKYH</font></font></font></p>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2"><b>WARNING:</b></font></font><font face="Arial, sans-serif"><font size="2">
</font></font><font face="Arial, sans-serif"><font size="2"><i><b>do
not introduce blank space into the header line</b></i></font></font><font face="Arial, sans-serif"><font size="2">.
For example, type </font></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">&gt;Red_kangaroo
or &gt;Redkangaroo, BUT </font></font><font face="Arial, sans-serif"><font size="2">NOT
</font></font><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">&gt;Red
kangaroo</font></font></font></font></font></p>
<p style="margin-bottom: 0in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="Arial, sans-serif"><font size="2">Do
not be concerned with blanks spaces or line breaks in the sequence
itself. These will be ignored.</font></font></font></p>
<p style="margin-left: 0.25in; margin-bottom: 0in; line-height: 0.18in;" align="justify" lang="en-US">
<br>
</p>
<p style="margin-bottom: 0in; line-height: 0.18in;" align="left" lang="en-US">
<font color="#000000"><font face="Arial, sans-serif"><font size="2">Use
the dropdown boxes at the bottom of the page to tell the application
what kind of data you are uploading. The Entity Type is &#8220;Protein&#8221;,
the Data Type is &#8220;Sequence&#8221;, and the format is &#8220;FASTA&#8221;. Now
select &#8220;SAVE&#8221;. You only need to set these once, the application
will remember after that.<br></font></font></font><br>
</p>
<ol start="5"><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">.
	Return to UniProt (on the other web page). Copy and Paste the
	remaining hemoglobin protein sequences into the NGBW in exactly the
	same way. You should have 5 data items in your data folder when you
	are finished. </font></font></font></font></font>
	</p>
</li></ol>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>
<p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><u><b>ALIGN
YOUR SEQUENCES </b></u><u><b><br></b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">Sequence
alignment is a tool evolutionary biologists use to assess how closely
related two sequences are. One tool for this is </font><u><b>ClustalW</b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">,
a software program that aligns protein and DNA sequences. </font></font></font></font>
</p>
<ol><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">To use <span style="font-weight: bold;">ClustalW</span> (and other software packages) in the NGBW, you
	create &#8220;<span style="font-weight: bold;">Tasks</span>.&#8221; Here&#8217;s how. C</font></font></font><font face="Arial, sans-serif"><font size="2">lick
	on the &#8220;<span style="font-weight: bold;">Tasks</span>&#8221; icon in your working folder. When the Task
	management page appears, click on the &#8220;<span style="font-weight: bold;">Create New Task</span>&#8221; Button.
	This will open a task creation window.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">Enter a description for the task you will recognize, and click the
	&#8220;<span style="font-weight: bold;">Set Description</span>&#8221; button.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">Now click on the &#8220;<span style="font-weight: bold;">Select Input Data</span>&#8221; button. Check the click
	boxes to the left of all the sequences you have entered for this
	exercise. Click the &#8220;<span style="font-weight: bold;">Select Data</span>&#8221; button.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">When the Task Creation Pane re-appears, click the &#8220;<span style="font-weight: bold;">Select Tool</span>&#8221;
	button. From the <span style="font-weight: bold;">Protein Sequence Tools tab</span>, choose &#8220;<span style="font-weight: bold;">CLUSTALW_P</span>&#8221;
	(the tools are alphabetical). Now click the "<span style="font-weight: bold;">Save and Run Task</span>"
	buttons.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">A new page will load that lets you follow the progress of your jobs.
	Click the &#8220;<span style="font-weight: bold;">Refresh Tasks</span>&#8221; tab near the top of the page, until
	the &#8220;<span style="font-weight: bold;">View Status</span>&#8221; button on the right turns into &#8220;View
	Results.&#8221; <span style="font-weight: bold;">Click on the &#8220;View Results</span>&#8221;: tab, and a page showing
	your results will appear. Click on the &#8220;<span style="font-weight: bold;">View</span>&#8221; button for
	&#8220;<span style="font-weight: bold;">outfile.aln</span>&#8221;.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">Examine the output file. It shows the calculated alignment, and the
	changes in amino acid identity at each hemoglobin Beta position as
	the species evolve. Save the alignment data, by clicking the &#8220;<span style="font-weight: bold;">Save
	to Current Folder</span>&#8221; button. Use the drop down boxes to set the
	Entity type to &#8220;<span style="font-weight: bold;">Protein</span>&#8221;, the Data type to &#8220;<span style="font-weight: bold;">Sequence
	Alignmen</span>t&#8221;, and the Format Type to &#8220;<span style="font-weight: bold;">Clustal</span>.&#8221;  Now click
	&#8220;<span style="font-weight: bold;">Save</span>&#8221;. </font></font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">&nbsp;It is easier to see which parts of the protein are well-conserved
	(unchanged) and which parts of the protein have experienced
	mutations by preparing a color coded diagram to emphasize the
	changes. To do this, you can create a task that uses the program
	<span style="font-weight: bold;">Boxshade</span>. </font></font></font><font face="Arial, sans-serif"><font size="2">Click
	the &#8220;<span style="font-weight: bold;">Create New Task</span>&#8221; button. Give the task a description and
	<span style="font-weight: bold;">&#8220;Set the Description</span>&#8221;, just like before. Click the &#8220;<span style="font-weight: bold;">Select
	Data</span>&#8221; button, and find your alignment data, check the box to the
	left of the alignment you created above (the .aln file), and click
	&#8220;<span style="font-weight: bold;">Select Data</span>.&#8221; When the task creation page reloads, choose
	Select Tool, and find &#8220;<span style="font-weight: bold;">BOXSHADE</span>&#8221; under the <span style="font-weight: bold;">Phylogeny/Alignment
	Tools tab</span>. Then click the &#8220;<span style="font-weight: bold;">Save</span>&#8221; button. When the task creation
	page re-appears, click the &#8220;<span style="font-weight: bold;">Save and Run</span>&#8221; button.</font></font></font></font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">.
	To view a color coded alignment, click on the &#8220;<span style="font-weight: bold;">Output</span>&#8221; button of
	this task, and click the &#8220;<span style="font-weight: bold;">View</span>&#8221; button for the file
	<b>boxshade.html</b>. The portions of the sequence that are
	conserved between all species will be highlighted in black. Amino
	acids that are similar, but not identical will be shown with a gray
	background. Those that have a different character will be shown with
	a white background. These represent areas of the amino acid sequence
	that have experienced (and tolerated) genetic changes. </font></font></font>
	</p>
</li></ol>
<p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="Arial, sans-serif"><font size="2">Take
a moment and look at the screen in front of you. Each line is the
amino acid sequence of the same protein (hemoglobin beta) in
different species. It is interesting to scan along the amino acid
sequences and look how they line up &#8212; how they are the same in the
different species and how they are different. You can see the traces
of evolutionary processes here: where amino acids have changed where
they have stayed the same, and where amino acids have been lost. You
are looking at the record of evolutionary history! </font></font></font>
</p>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">Take
a screenshot of this alignment chart to use in your lab report (press
the &#8220;Printscreen&#8221; key, typically labeled &#8220;PrtScn&#8221;). The
picture of the screen is now waiting to be pasted into a document.
Paste the screenshot into a Word or Open Office document to use later. </font></font></font>
</p>
<p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><br></font></font><u><b>DETERMINE
THE DISTANCE BETWEEN SEQUENCE PAIRS</b></u></font></font></font></p>
<ol><li><p style="margin-top: 0.07in; margin-bottom: 0.07in; line-height: 100%;">
	<font face="Arial, sans-serif"><font size="2">While your eye can
	tell qualitatively what has occurred between species, rigorous
	analysis requires that we quantify these differences. The next step
	is to quantify the evolutionary distance between sequence pairs. To
	do this, go back to the Tasks area of your folder. Click the <b>&#8220;Create
	New Task</b>&#8221; button. Give the task a description and <b>&#8220;Set
	the Description&#8221;</b>, just like before. Click the &#8220;<b>Select
	Data</b>&#8221; button, and find your alignment data, check the box to
	the left of the alignment you created above (the .aln file), and
	click &#8220;<b>Select Data</b>.&#8221; When the task creation page reloads,
	choose <b>Select Too</b>l, and find &#8220;<b>CLUSTALW_DIST&#8221;</b> under
	the <b>Phylogeny/Alignment Tools</b> tab. Then click the <b>&#8220;Save</b>&#8221;
	button. When the task creation page re-appears, click the <b>&#8220;Save
	and Run&#8221;</b> button. </font></font>
	</p>
	</li><li><p style="margin-top: 0.07in; margin-bottom: 0.07in; line-height: 100%;">
	<font face="Arial, sans-serif"><font size="2">When the Task
	management page reloads, use the &#8220;<b>Refresh Tasks</b>&#8221; button
	to monitor when the job completes. When the &#8220;<b>View Output</b>&#8221;
	button appears, click on it, and expose the results. Click on the
	<b>View</b> button for the <b>&#8220;infile.dst&#8221;</b> link to expose
	the Distance Matrix results. Record the distance matrix to examine
	later.</font></font></p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font face="Arial, sans-serif"><font size="2">The
	CLUSTALW_DIST program also produces a phylogenetic tree inferred
	from this data. TO view this tree return to the</font></font><font face="Arial, sans-serif"><font size="2"><b>
	Results</b></font></font><font face="Arial, sans-serif"><font size="2">
	page, and click the &#8220;</font></font><font face="Arial, sans-serif"><font size="2"><b>View</b></font></font><font face="Arial, sans-serif"><font size="2">&#8221;
	button for the file &#8220;</font></font><font face="Arial, sans-serif"><font size="2"><b>infile.ph</b></font></font><font face="Arial, sans-serif"><font size="2">&#8221;
	You will see file containing the names of your sequences, separated
	by colons and parentheses. You can&#8217;t interpret this result
	directly, but the NGBW provides an application to do that for you.
	Click the &#8220;</font></font><font face="Arial, sans-serif"><font size="2"><b>Save
	to Current Folder</b></font></font><font face="Arial, sans-serif"><font size="2">&#8221;
	button. Use the drop down boxes to set the </font></font><font face="Arial, sans-serif"><font size="2"><b>Entity
	type to &#8220;Taxon</b></font></font><font face="Arial, sans-serif"><font size="2">&#8221;,
	the </font></font><font face="Arial, sans-serif"><font size="2"><b>Data
	type to &#8220;Phylogenetic Tree</b></font></font><font face="Arial, sans-serif"><font size="2">&#8221;,
	and the </font></font><font face="Arial, sans-serif"><font size="2"><b>Format
	Type to &#8220;Newick</b></font></font><font face="Arial, sans-serif"><font size="2">.&#8221;
	Now click &#8220;</font></font><font face="Arial, sans-serif"><font size="2"><b>Save</b></font></font><font face="Arial, sans-serif"><font size="2">&#8221;.</font></font></font></font></font></p>
</li></ol>
<p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><b>VIEW
THE TREE</b></font></font></font></p>
<ol><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">To
	view the tree in a human-interpretable form, </font><font face="NFKRIX+ArialMT, Arial, sans-serif"><b>return
	to the Data area of your folder,</b></font><font face="NFKRIX+ArialMT, Arial, sans-serif">
	and click the tab &#8220;</font><font face="NFKRIX+ArialMT, Arial, sans-serif"><b>Phylogenetic
	Trees</b></font><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;.
	Click the Label corresponding to the phylogenetic tree you created,
	and it will open up, revealing these two links &#8220;Show/Hide Data
	Contents | Draw Tree.&#8221; Cllck on the &#8220;</font><font face="NFKRIX+ArialMT, Arial, sans-serif"><b>Draw
	Tree</b></font><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;
	link, and you will see an interactive view of the Tree. </font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0.2in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">.
	In the interpretation of the phylogenetic tree, it is useful to
	&#8220;root&#8221; the tree using a more distant relative of the add one
	more step to make our tree more accurate. We need to add an
	&#8220;</font><u><b>outgroup</b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;
	to the mix of species we are analyzing. An </font><font face="NFKRIX+ArialMT, Arial, sans-serif"><b>outgroup</b></font><font face="NFKRIX+ArialMT, Arial, sans-serif">
	clarifies the evolutionary relationships by providing a &#8220;root&#8221;
	to the tree. It is designated as a species that is far removed from
	any of the other species we are comparing. Here, the red kangaroo
	was included for this purpose, since it is a marsupial in contrast
	to all the other mammals in our study, which are all placentals.
	Therefore, the kangaroo is selected to be the </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>most
	different </i></font><font face="NFKRIX+ArialMT, Arial, sans-serif">organism
	from the other mammals on your tree. Redraw the tree with red
	kangaroo as the outgroup, by entering the name you entered after the
	&gt; mark on the header file of your fasta sequence corresponding to
	red kangaroo. If you followed this exercise exactly as above, </font><font face="NFKRIX+ArialMT, Arial, sans-serif"><b>enter
	red_kangaroo, and click &#8220;Redraw</b></font><font face="NFKRIX+ArialMT, Arial, sans-serif">&#8221;.
	If you get the message &#8220;None of the outgroup taxa are valid&#8221; it
	means you mistyped the outgroup name, or you forgot the name you
	chose. The case of the characters is important, and must also match
	exactly. Record a screen shot in your lab report.</font></font></font></font></p>
</li></ol>
<ol><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">Question
	#1  What conclusions do you come to about the evolutionary
	relationship amongst seals, whales, dogs, and cows?<br>NOTE:
	Phylogenetic trees built with this software can only be used to make
	conclusions about common ancestry. They cannot be used to make
	conclusions about the timeframe of evolution. The length of branches
	is not a measure of evolutionary time. It is merely an artifact of
	physically arranging the tree.</font></font></font></p></ol>

<p style="margin-bottom: 0in; page-break-before: auto; page-break-after: auto;">
<font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 11pt;" size="3"><b>PART
B </b></font></font>
</p>
<ol><li><p style="line-height: 0.18in;" align="justify" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">.
	Below are the representative marine mammals for your new cladogram</font></font></font></p>
</li></ol>
<ul><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">minke
	whale (baleen whales) </font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">dolphin
	(toothed whales) </font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">harbor
	seal </font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">walrus
	</font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">otter
	</font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">manatee</font></font></font></p>
</li></ul>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>
<ol start="2"><li><p style="line-height: 0.18in;" align="justify" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">.
	In this exercise, we will compare the hemoglobin proteins of these
	marine mammals to some representatives of the major taxa of land
	mammals: </font></font></font>
	</p>
</li></ol>
<ul><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">carnivora:
	dog, </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Canis
	familiaris </i></font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">rodentia:
	rat, </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Rattus
	norvegicus </i></font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">herbivore:
	cow, </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Bos
	taurus </i></font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">primates:
	human </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Homo
	sapiens </i></font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">proboscidea:
	African elephant, </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Loxodonta
	africana </i></font></font></font></font>
	</p>
	</li><li><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">marsupials:
	red kangaroo, </font><font face="JZJNGM+Arial-ItalicMT, Arial, sans-serif"><i>Macropus
	rufus </i></font></font></font></font>
	</p>
</li></ul>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>

<p style="margin-left: 0.25in; text-indent: -0.25in; margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">Reminder:
We are testing the hypothesis that all marine mammals have a common
land mammal ancestor.
</font></font></font>
</p>
<p style="margin-left: 0.25in; text-indent: -0.25in; margin-bottom: 0in;" align="left" lang="en-US">
<br>
</p>
<ol start="3"><li><p style="line-height: 0.18in;" align="justify" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">Once again, use the hemoglobin beta protein sequence to test this
	hypothesis. You will be building a tree with </font><u><b>all </b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">the
	marine mammals and </font><u><b>all </b></u><font face="NFKRIX+ArialMT, Arial, sans-serif">the
	land mammals listed above. </font></font></font></font>
	</p>
	</li><li><p style="line-height: 0.18in;" align="justify" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">Follow the complete process we used in Part A to obtain your amino
	acid sequences from UniProt, align them in ClustalW, and build the
	evolutionary tree in the NGBW. Take a screenshot of your aligned
	amino acid sequences and your phylogenetic tree and save them in a
	Word document to be included in your lab write up.</font></font></font></font></p>
	</li><li><p style="line-height: 0.18in;" align="justify" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><font face="NFKRIX+ArialMT, Arial, sans-serif">Then use your phylogenetic tree to determine the evolutionary
	relationships amongst marine mammals and the representative land
	mammals. Determine whether your hypothesis was supported by the
	molecular data. </font></font></font></font>
	</p>
</li></ol>
<p style="page-break-before: always;" align="justify" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font style="font-size: 11pt;" size="3"><b>Write
up requirements:</b></font></font></font></font></p>
<p align="justify" lang="en-US"><font color="#000000"><font face="RJXDLZ+Arial-BoldMT, Arial, sans-serif"><font size="3"><font style="font-size: 11pt;" size="3"><b>D</b></font><font style="font-size: 9pt;" size="2"><b>ISCUSSION
</b></font></font></font></font>
</p>
<p style="margin-bottom: 0.25in; line-height: 0.18in;" align="justify" lang="en-US">
<font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">In
addition to Question #1 above, use the following questions to develop
your discussion of the results. (make sure to attach all screen shots
or copy and pastes of all distance matrices.  Hand record on your
trees the percent differences between the species using the distance
matrices data).</font></font></font></p>
<ol><p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">2.
	Did your tree support your hypothesis? Explain. </font></font></font>
	</p><p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">3.
	What does this phylogenetic tree structure suggest about the
	evolutionary history of marine mammals? Go into detail here about
	what parts of the tree lead you to what conclusions about the
	evolutionary history of the marine mammals. </font></font></font>
	</p><p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">4.
	If marine mammals share common morphological characteristics, what
	do your conclusions about their evolutionary history imply about
	these common characteristics? </font></font></font>
	</p><p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">5.
	What is the advantage of using the protein sequence from the
	hemoglobin beta gene to prepare the comparisons between species? </font></font></font>
	</p><p style="margin-bottom: 0.21in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">6.
	Why do you need to align the sequences (with ClustalW) before
	inputting them into CLUSTAL_DIST? </font></font></font>
	</p><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">7.
	What organism served as your outgroup? Why? What function does the
	outgroup serve? </font></font></font>
	</p></ol>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><br>
</p>
<p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2">Include
references.</font></font></font></p><p style="margin-bottom: 0in;" align="left" lang="en-US"><font color="#000000"><font face="NFKRIX+ArialMT, Arial, sans-serif"><font style="font-size: 10pt;" size="2"><img style="height: 486px; width: 290px;" alt="whale evolution" src="file:///C:/Users/rgroch/Pictures/whale-evolution.jpg"></font></font></font></p></body></html>